regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -47,7 +47,8 @@ var CommentReviewView = Backbone.View.extend({
 
   editComment: function(e) {
     var section = $(e.target).closest('li').data('section');
-    var options = {id: section, section: section, mode: 'write'};
+    var label = $(e.target).closest('li').find('.comment-section-label').data('section-label');
```

Comment:
Would it make sense to just use `text()` instead? Seems like we're not gaining anything with the `data` attribute. 

If we should keep it, what do you think of renaming it `label` and hang it on the `li` for parity?